### ISSUE TITLE
strange warning during training

### ISSUE THREAD

**Issue by krzysztoffiok** [zalandoresearch/flair#1244]
When i started training on a new machine today i got this warning between epochs, just before starting the next epoch after printing scores on dev. Also, it appears only after the first epoch, after the 2nd and later there is no warning.

2019-10-24 17:14:30,646 epoch 1 - iter 540/546 - loss 3.11813573 - samples/sec: 44.61
2019-10-24 17:14:31,606 
2019-10-24 17:14:31,607 EPOCH 1 done: loss 3.1156 - lr 0.2000
2019-10-24 17:14:55,697 DEV : loss 2.5082967281341553 - score 0.1624
2019-10-24 17:14:55,833 BAD EPOCHS (no improvement): 0
/home/____/env/nlp/lib/python3.7/site-packages/torch/serialization.py:292: UserWarning: Couldn't retrieve source cod
e for container of type DocumentRNNEmbeddings. It won't be checked for correctness upon loading.
  "type " + obj.__name__ + ". It won't be checked "

Flair 0.4.3 ubuntu 19.10, pytorch-transformers 1.2.0

**Comment by gcp** [comment]
Are you on PyTorch 1.3? 

https://github.com/pytorch/pytorch/issues/27972

**Comment by krzysztoffiok** [comment]
Hi,
this is my pip freeze output

atomicwrites==1.3.0
attrs==19.3.0
backcall==0.1.0
boto==2.49.0
boto3==1.10.1
botocore==1.13.1
bpemb==0.3.0
bson==0.5.8
certifi==2019.9.11
chardet==3.0.4
Click==7.0
cloudpickle==1.2.2
cycler==0.10.0
decorator==4.4.0
Deprecated==1.2.6
docutils==0.15.2
et-xmlfile==1.0.1
flair==0.4.3
future==0.18.1
gensim==3.8.1
hyperopt==0.2.1
idna==2.8
importlib-metadata==0.23
ipython==7.6.1
ipython-genutils==0.2.0
jdcal==1.4.1
jedi==0.15.1
jmespath==0.9.4
joblib==0.14.0
kiwisolver==1.1.0
langdetect==1.0.7
matplotlib==3.1.1
more-itertools==7.2.0
mpld3==0.3
networkx==2.2
numpy==1.17.3
openpyxl==3.0.0
packaging==19.2
pandas==0.25.2
parso==0.5.1
pexpect==4.7.0
pickleshare==0.7.5
pkg-resources==0.0.0
pluggy==0.13.0
prompt-toolkit==2.0.10
ptyprocess==0.6.0
py==1.8.0
Pygments==2.4.2
pyparsing==2.4.2
pytest==5.2.1
python-dateutil==2.8.0
pytorch-transformers==1.2.0
pytz==2019.3
regex==2019.8.19
requests==2.22.0
s3transfer==0.2.1
sacremoses==0.0.35
scikit-learn==0.21.3
scipy==1.3.1
segtok==1.5.7
sentencepiece==0.1.83
six==1.12.0
sklearn==0.0
smart-open==1.8.4
sqlitedict==1.6.0
tabulate==0.8.5
torch==1.3.0
tqdm==4.36.1
traitlets==4.3.3
urllib3==1.24.3
wcwidth==0.1.7
wrapt==1.11.2
xlrd==1.2.0
zipp==0.6.0

**Comment by gcp** [comment]
>torch==1.3.0

You're probably seeing the upstream bug linked above.

**Comment by krzysztoffiok** [comment]
Ok thanks for explaining!
